doc/ExpertOptions.md

```diff
@@ -106,6 +108,26 @@ tasks by default.
 $ java -jar dxWDL-0.44.jar compile test/files.wdl -project project-xxxx -defaults test/files_input.json -extras extraOptions.json
 ```
 
+## Describe WDL workflow to obtain execution tree
+
+You can also describe a dnanexus workflow that was compiled by dxWDL to get a execution tree presentating the workfow.
```

Comment:
You can describe a dnanexus workflow that was compiled by dxWDL to get an execution tree presentating the workfow.